wait_for_local_port.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+wait_for_local_port() {
```

Comment:
I simply reused the script from Integration, which has been around for a while already :)
Also, I think that writing Node.js for this would end up being much messier/complex.